client/my-sites/people/people-invite-details/index.jsx

```diff
@@ -0,0 +1,167 @@
+	};
+
+	renderClearOrRevoke = () => {
+		const { deleting, invite, translate } = this.props;
+		const { isPending } = invite;
+		const revokeMessage = translate(
+			'Revoking an invite will no longer allow this person to join your site. ' +
+				'You can always invite them again if your change your mind.'
+		);
+		const clearMessage = translate(
+			'If you no longer wish to see this record, you can clear it. ' +
+				'The person will still remain a member of this site.'
+		);
+
+		return (
+			<div className="people-invite-details__clear-revoke">
+				<div>{ isPending ? revokeMessage : clearMessage }</div>
+				<Button
+					busy={ deleting }
+					primary={ isPending }
+					scary={ isPending }
+					onClick={ this.handleDelete }
+				>
+					{ isPending ? translate( 'Revoke Invite' ) : translate( 'Clear Invite' ) }
+				</Button>
+			</div>
+		);
+	};
+
+	renderInvite = () => {
+		const { site, invite } = this.props;
+
+		return (
+			<div>
+				<Card>
+					<PeopleListItem
+						key={ invite.key }
+						invite={ invite }
+						user={ invite.user }
+						site={ site }
+						type="invite-details"
+						isSelectable={ false }
+					/>
+					{ this.renderInviteDetails() }
+				</Card>
+				{ this.renderClearOrRevoke() }
+			</div>
+		);
+	};
+
+	renderInviteDetails = () => {
+		const { invite, translate, moment } = this.props;
+		const showName = invite.invitedBy.login !== invite.invitedBy.name;
+
+		return (
+			<div className="people-invite-details__meta">
+				<div className="people-invite-details__meta-item">
+					<span className="people-invite-details__meta-item-label">
+						{ translate( 'Invited By' ) }
+					</span>
+					<Gravatar user={ invite.invitedBy } size={ 24 } />
+					{ showName && (
+						<span className="people-invite-details__meta-item-user">{ invite.invitedBy.name }</span>
+					) }
+					<span className="people-invite-details__meta-item-username">
+						{ '@' + invite.invitedBy.login }
+					</span>
+				</div>
+				<div className="people-invite-details__meta-item">
+					<span className="people-invite-details__meta-item-label">{ translate( 'Sent' ) }</span>
+					<span className="people-invite-details__meta-item-date">
+						{ moment( invite.inviteDate ).format( 'LLL' ) }
+					</span>
+				</div>
+				{ invite.acceptedDate && (
+					<div className="people-invite-details__meta-item">
+						<span className="people-invite-details__meta-item-label">
+							{ translate( 'Accepted' ) }
+						</span>
+						<span className="people-invite-details__meta-item-date">
+							{ moment( invite.acceptedDate ).format( 'LLL' ) }
+						</span>
+					</div>
+				) }
+			</div>
+		);
+	};
+
+	render() {
+		const { site, translate, invite } = this.props;
+
+		if ( ! site || ! site.ID ) {
+			return null;
+		}
+
+		return (
+			<Main className="people-invite-details">
+				<QuerySiteInvites siteId={ site.ID } />
+				<SidebarNavigation />
+
+				<HeaderCake isCompact onClick={ this.goBack }>
+					{ translate( 'Invite' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **60** times:
`translate( 'Invites' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).